src/components/Projects/Project.tsx

```diff
@@ -18,6 +20,7 @@ export default function Project({
   link,
 }: ProjectProps) {
   const [data, setData] = useState<RepoData | null>(null);
+  const [languagesData, setLanguagesData] = useState<LanguageInfo | null>(null);
 
   useEffect(() => {
     function getRepo() {
```

Comment:
Sorry, I have also made a mistake here by not making this function async, sorry for that.

Reply:
Yes no problem, I'm going to improve the api calls, I think I can have it ready in 2-3 days